controllers/bsl_test.go

```diff
@@ -1095,6 +1122,7 @@ func TestDPAReconciler_ValidateBackupStorageLocations(t *testing.T) {
 					Name:      "cloud-credentials",
 					Namespace: "test-ns",
 				},
+				Data: map[string][]byte{"cloud": []byte("dummy_data"), "testing": []byte("dummy_data2")},
```

Comment:
What's the `testing` key here for?

Reply:
haha, can be removed, took this yaml from validator_test file. let me make it "cloud"

Reply:
fixed.

Reply:
ahh sorry. didn't notice the keys were specified in cloudstorage

Reply:
before was probably better if there's a check for `dummy_data2` being used correctly..

Reply:
but since there's not, up to you.